utils/bisect.js

```diff
@@ -49,23 +53,26 @@ if (argv.h || argv.help) {
 }
 
 if (typeof argv.good !== 'number') {
-  console.log(
-    COLOR_RED + 'ERROR: expected --good argument to be a number' + COLOR_RESET
-  );
-  console.log(help);
-  process.exit(1);
+  argv.good = getChromiumRevision();
```

Comment:
should not we still report errors if an invalid value is provided? e.g., --good v232323